### ISSUE TITLE
Can we expose an url to article.pdf and source_code.zip?

### ISSUE THREAD

**Issue by phcerdan** [InsightSoftwareConsortium/InsightJournal#75]
I am building the XML that we are going to handle to Crossref for them to register.
There is a `<resource>` tag where we can directly link an url to the `article.pdf` or the source code associated to the publication.

I wonder if we have an exposed url to directly that file?

I see that when the user press the button `Download PDF` on the article tab it downloads it, but not sure if there is an url.

@thewtex

**Comment by phcerdan** [comment]
> @phcerdan For the CrossRef XML, does it support revision specification?
 Do you mean this set of headers on the xml? The following is taken from the template.xml we use.
 
 ```xml
<?xml version="1.0" encoding="UTF-8"?>
<doi_batch xmlns="http://www.crossref.org/schema/4.4.2" xmlns:ai="http://www.crossref.org/AccessIndicators.xsd" xmlns:rel="http://www.crossref.org/relations.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="4.4.2" xsi:schemaLocation="http://www.crossref.org/schema/4.4.2 http://www.crossref.org/schemas/crossref4.4.2.xsd">
```

**Comment by phcerdan** [comment]
Fixed in #86 :tada:
